ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py

```diff
@@ -24,6 +24,7 @@
 from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
+from launch.actions import OpaqueFunction
```

Comment:
can we here somehow reuse at least part of the`ur_moveit.launch.py`-file? This huge launch files are getting un-maintanable.